<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VC Patcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="PathE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseE">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseR">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="PathR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>281</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Inject">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="romType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="currentText">
     <string>NES</string>
    </property>
    <item>
     <property name="text">
      <string>NES</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SNES LoROM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SNES HiROM</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="start">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0x10064C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="end">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>0x14065B</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_ROM"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEnglish">
    <property name="title">
     <string/>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionSpanish"/>
    <addaction name="actionItalian"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEnglish"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="fileopen">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionOpen_ROM">
   <property name="icon">
    <iconset theme="fileopen">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Open ROM...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="filesave">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Inject...</string>
   </property>
   <property name="toolTip">
    <string>Inject</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/flags/uk.png</normaloff>:/images/flags/uk.png</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionItalian">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/flags/it.png</normaloff>:/images/flags/it.png</iconset>
   </property>
   <property name="text">
    <string>Italiano</string>
   </property>
  </action>
  <action name="actionSpanish">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/flags/es.png</normaloff>:/images/flags/es.png</iconset>
   </property>
   <property name="text">
    <string>Espa√±ol</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
